unicodetools/data/idna/13.1.0/ReadMe.txt

```diff
@@ -0,0 +1,10 @@
+# Unicode IDNA Mapping and Test Data
+# Date: 2019-10-07, 17:10:00 GMT
+# © 2019 Unicode®, Inc.
+# Unicode and the Unicode Logo are registered trademarks of Unicode, Inc. in the U.S. and other countries.
+# For terms of use, see http://www.unicode.org/terms_of_use.html
+
+This directory contains DRAFT data files for version 13.0.0 of
```

Comment:
Looks like "fake" copies of 13.0 files into a 13.1 folder. There was no Unicode 13.1, only emoji 13.1. If the JSP code currently requires fake dot-version files, then please create a bug/issue to fix that in the future, so we need not do this again.

Reply:
@macchiati ?

Reply:
https://github.com/unicode-org/unicodetools/issues/100 tnx